integrations/dialpad/syncs/users.ts

```diff
@@ -0,0 +1,34 @@
+import type { NangoSync, ProxyConfiguration } from '../models';
+import type { DialpadUser, User } from '../types';
+
+export default async function fetchData(nango: NangoSync) {
+    
+    const proxyConfiguration: ProxyConfiguration = {
+ 
+        // https://developers.dialpad.com/reference/userslist
+        endpoint: '/users',
+        paginate: {
+            type: 'cursor',
+            cursor_name_in_request: 'cursor',
+            cursor_path_in_response: 'next_cursor',
+            response_path: 'users',
```

Comment:
The dialpad documentation says the response path will be `items`
<img width="659" alt="image" src="https://github.com/user-attachments/assets/5143e8b3-6a86-49cd-bab5-b31bfbd45c12">